engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -97,7 +97,7 @@
     private LocalPlayer localPlayer;
     @In
     private WorldProvider worldProvider;
-    private Camera playerCamera;
+    public Camera playerCamera;
```

Comment:
Yeah, that would be better. But `@In` annotation doesn't work on `WorldRendererImpl`. I think i will need to get it directly from context. Will make the changes today :)